jsonnet/rules.libsonnet

```
@@ -428,6 +428,15 @@ function(params) {
           record: 'cluster:usage:workload:kube_running_pod_ready:avg',
           // Report the percentage (0-1) of pending or running workload (everything outside of openshift-*) pods reporting ready
         },
+        {
+          expr: 'last_over_time(kube_pod_status_unschedulable[5m]) == 1',
```

To be consistent with KubePodNotReady
```suggestion
          expr: 'last_over_time(kube_pod_status_unschedulable{namespace=~"(openshift-.*|kube-.*|default)"}[5m]) == 1',
```